### PR TITLE
Minor text changes in about page

### PR DESCRIPTION
**Pull request by usr110**
Fixed a minor typo: `equivlaent` - missed it in a [PR](https://github.com/npct/pct-shiny/pull/754). Also added a space after a full stop.

### CODE REVIEW

File: regions_www/tabs/about_body.html [hunk 1]
```diff
@@ -174,8 +174,8 @@ <h5>With many thanks also to former PCT team members <b>Alvaro Ullrich</b> and <
...
-         <p> First, the PCT is a strategic planning tool. Different visions of the future are represented through various scenarios of change, including the Department for Transport’s draft <a href="http://www.gov.uk/government/uploads/system/uploads/attachment_data/file/364791/141015_Cycling_Delivery_Plan.pdf" target="_blank">Cycling Delivery Plan</a> target to double cycling in a decade and the more ambitious ‘Go Dutch’ scenario, whereby cycling levels equivlaent to the Netherlands are reached in England and Wales (allowing for English and Welsh hilliness and trip distances). By showing what the rate of cycling could feasibly look like in different parts of cities and regions, and illustrating the associated increase in cycle use on the road network, the PCT should inform policies that seek a wider shift towards sustainable transport.</p>
+         <p>The PCT was designed to assist transport planners and policy makers to prioritise investments and interventions to promote cycling. The PCT answers the question: 'where is cycling currently common and where does cycling have the greatest potential to grow?'. The PCT can be used at different scales.</p>
```
Comment (Robinlovelace) [inline regions_www/tabs/about_body.html]: Good fix.

File: regions_www/tabs/about_body.html [hunk 2]
```diff
@@ -14,7 +14,7 @@
...
-        <p> This is the online home of the open source transport planning system, the Propensity to Cycle Tool (PCT).The PCT is released under the <a href="/www/licence.html" target="_blank">Affero GPL</a>: it is free to use, copy and modify, e.g. to create versions for new cities and states. Please cite the journal article Lovelace et al. (<a href="https://www.jtlu.org/index.php/jtlu/article/view/862" target="_blank">2017</a>) and/or Goodman et al. (<a href="https://www.sciencedirect.com/science/article/pii/S2214140518301257" target="_blank">2019</a>) if you use the PCT in your work.</p>
+        <p> This is the online home of the open source transport planning system, the Propensity to Cycle Tool (PCT). The PCT is released under the <a href="/www/licence.html" target="_blank">Affero GPL</a>: it is free to use, copy and modify, e.g. to create versions for new cities and states. Please cite the journal article Lovelace et al. (<a href="https://www.jtlu.org/index.php/jtlu/article/view/862" target="_blank">2017</a>) and/or Goodman et al. (<a href="https://www.sciencedirect.com/science/article/pii/S2214140518301257" target="_blank">2019</a>) if you use the PCT in your work.</p>
```
Comment (Robinlovelace) [inline regions_www/tabs/about_body.html]: I cannot see any change here.

File: regions_www/tabs/about_body.html [hunk 3]
```diff
@@ -174,8 +174,8 @@ <h5>With many thanks also to former PCT team members <b>Alvaro Ullrich</b> and <
...
+         <p>The PCT was designed to assist transport planners and policy makers to prioritise investments and interventions to promote cycling. The PCT answers the question: 'where is cycling currently common and where does cycling have the greatest potential to grow?'. The PCT can be used at different scales.</p>
+         <p> First, the PCT is a strategic planning tool. Different visions of the future are represented through various scenarios of change, including the Department for Transport’s draft <a href="http://www.gov.uk/government/uploads/system/uploads/attachment_data/file/364791/141015_Cycling_Delivery_Plan.pdf" target="_blank">Cycling Delivery Plan</a> target to double cycling in a decade and the more ambitious ‘Go Dutch’ scenario, whereby cycling levels equivalent to the Netherlands are reached in England and Wales (allowing for English and Welsh hilliness and trip distances). By showing what the rate of cycling could feasibly look like in different parts of cities and regions, and illustrating the associated increase in cycle use on the road network, the PCT should inform policies that seek a wider shift towards sustainable transport.</p>
```
Comment (Robinlovelace) [inline regions_www/tabs/about_body.html]: Another good fix.

File: regions_www/tabs/about_body.html (hunk 2 above)
Comment (usr110) [reply regions_www/tabs/about_body.html]: @Robinlovelace 
As the commit says, I've added a space. Find it just before `The PCT is released under the  <a href="/www/licence.html" target="_blank">Affero GPL`

Comment (Robinlovelace) [reply regions_www/tabs/about_body.html]: Ah yes. Cheers.

**Pull request merged**